products/apache-apisix.md

```diff
@@ -0,0 +1,84 @@
+---
+title: Apache Apisix
+category: app
+permalink: /apisix
+alternate_urls:
+-   /apache-apisix
+
+auto:
+  methods:
+  -   git: https://github.com/apache/apisix
+
+identifiers:
+  - repology: apisix
+
+releases:
+-   releaseCycle: "3.9"
+    releaseDate: 2024-03-25
+    eol: false
+    lts: false
+    latest: "3.9.1"
+    latestReleaseDate: 2024-04-24
+    link: https://github.com/apache/apisix/blob/master/CHANGELOG.md#390
```

Comment:
Linking to GitHub release is easier / less subject to break. You could use the following `changelogTemplate` (more info on https://endoflife.date/contribute) to do that easily for all releases:

```
changelogTemplate: https://github.com/apache/apisix/releases/tag/__LATEST__
```

Reply:
Done.